modules/osm/tags.js

```diff
@@ -373,3 +373,9 @@ export var osmSummableTags = new Set([
     'parking:left:capacity',
     'parking:left:capacity'
 ]);
+export  const DASH_SENSITIVE_KEYS = new Set([
```

Comment:
we typically use camel casing for exported constants:

```suggestion
export  const dashSensitiveKeys = new Set([
```